src/main/scala/it/pagopa/interop/apigateway/api/impl/package.scala

```diff
@@ -197,4 +201,23 @@ package object impl extends SprayJsonSupport with DefaultJsonProtocol {
       case AttributeRegistryManagementApiAttributeKind.VERIFIED  => AttributeKind.VERIFIED
     }
   }
+
+  implicit class EnrichedMessage(private val messages: NotifierApiMessages) extends AnyVal {
+    def toModel: Try[Messages] =
+      Success(
+        Messages(
+          limit = messages.limit,
+          size = messages.size,
+          nextId = messages.nextId,
+          messages = messages.messages.map(toMessageModel)
+        )
+      )
+
+    def toMessageModel(message: NotifierApiMessage): Message = Message(
```

Comment:
this function has not a real reason to be in this implicit class.
e.g. I would not expect to call `messages.toMessageModel(???)` as it does not have any relation with `messages` object